server/gendb/table.go.tmpl

```diff
@@ -883,6 +886,87 @@ func (t {{ $modelName }}Table) scan{{ $modelName }}sBy{{ $computedIndexName }}(c
   return err
 }
 {{- end }}
+
+{{- if transactionsEnabled $xdbConfig }}
+{{- range $transactWithSchemaName := $xdbConfig.EnableTransactions }}
+{{- $transactWithModelName := pascalize $transactWithSchemaName }}
+func (t {{ $modelName }}Table) transactSave{{ $modelName }}And{{ $transactWithModelName }}(ctx context.Context, m1 models.{{ $modelName }}, m1ConditionMap map[string]interface{}, m2 models.{{ $transactWithModelName }}, m2ConditionMap map[string]interface{}) error {
+  data1, err := encode{{ $modelName }}(m1)
+  if err != nil {
+    return err
+  }
+
+  m1CondExpr, m1ExprVals, m1ExprNames, err := buildCondExpr(m1ConditionMap)
+  if err != nil {
+    return err
+  }
+
```

Comment:
An [example](https://github.com/Clever/messaging-service/pull/214/files#diff-3672c1b365f00721e784d7201508ceaa44053b758091d5aaf4adf7e191ec89c6R90-R94) of how the conditions can be used